src/remembrall/classes/Record.py

```diff
@@ -35,6 +37,29 @@ def add_birthday(self, birthday):
         if not self.birthday:
             self.birthday = Birthday(birthday)
 
+    def add_address(self, address):
+        if self.address:
+            return "Address already exists."
+        if address:
+            Address.check_address(address)
+            self.address = Address(address)
+        return "Address added"
+
+    def edit_address(self, new_address):
+        if new_address:
+            Address.check_address(new_address)
+            if isinstance(self.address, Address):
+                self.address.edit_address(new_address)
+                return "Address updated"
+            else:
+                return "No address to edit."
+    
+    def delete_address(self):
+        if isinstance(self.address, Address):
```

Comment:
тут можна не перевіряти на інстанс, можно просто перевірити чи існує. Бо щоб там не було, нам треба це видалити
```suggestion
        if self.address:
```

Reply:
Добре, дякую, буде зроблено.